### ISSUE TITLE
new words not updating 

### ISSUE THREAD

**Issue by bhavy3103** [abhijivani3001/multi-monkey#1]
![image](https://github.com/user-attachments/assets/13209bfa-0e46-4eef-b916-9f7d8b8c7240)

although time remains I am not able to see new words.

**Comment by shangesh-tech** [comment]
yes , i will do it , assign this to me

@bhavy3103 

can explain little bit more about this issue...

**Comment by bhavy3103** [comment]
> yes , i will do it , assign this to me
> 
> @bhavy3103
> 
> can explain little bit more about this issue...

@shangesh-tech  once you reach complete all the words shown on your screen and if time is remain then automatically new line will shown on screen which is not showing right now so we have to resolve this bug.

**Comment by shangesh-tech** [comment]
@bhavy3103 
Yes you are correct , i also face same issue , can you assign this to me i will solve this

![Screenshot 2024-10-15 135836](https://github.com/user-attachments/assets/03e97893-7c57-494d-83f2-d0a46430d6ab)

**Comment by bhavy3103** [comment]
I'm interested in solving this issue.
